tests/unit/Term/AliasGroupListTest.php

```diff
@@ -191,12 +184,10 @@ public function testGivenEmptyGroup_setGroupRemovesGroup() {
 			new AliasGroup( 'en', array( 'foo' ) ),
 		) );
 
-		$expectedList = new AliasGroupList();
-
 		$list->setGroup( new AliasGroup( 'en' ) );
 		$list->setGroup( new AliasGroup( 'de' ) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertTrue( $list->isEmpty() );
```

Comment:
We do this on many places. Do you think we should add `equals` to all objects for which we do and use that instead?

Another difference is that when the new assertion here fails, you'll get "failed asserting that false is true", rather than a diff.